src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -362,6 +362,10 @@
     "NAME": "Miracle",
     "DESCRIPTION": "Retain. NL Gain !M! [E] . NL Add !stsjorbsmod:MetaMagic! stsjorbsmod:Manifest."
   },
+  "stsjorbsmod:Mirror": {
+    "NAME": "Mirror",
+    "DESCRIPTION": "stsjorbsmod:Ephemeral. NL Until your next turn, take no damage. NL All enemies attack themselves."
```

Comment:
I think the `All enemies attack themselves` wording is maybe a little off; my reading of it would imply that it should retarget non-damage parts of attacks with power side effects, too, but I don't think that's going to happen. Maybe `Until your next turn, enemies that would damage you instead damage themselves.` or something?

Reply:
Actually, the original wording from the sheet seems fine here (`Until your next turn, you take no damage and enemies deal damage to themselves.`)